### PR TITLE
PE-6147: feat(empty state): implement empty state

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/6vbacts030lto

### CODE REVIEW

File: lib/pages/drive_detail/components/drive_detail_folder_empty_card.dart [hunk 1]
```diff
@@ -1,87 +1,704 @@
 part of '../drive_detail_page.dart';
 
-class DriveDetailFolderEmptyCard extends StatelessWidget {
+class DriveDetailFolderEmptyCard extends StatefulWidget {
```
Comment (kunstmusik) [inline lib/pages/drive_detail/components/drive_detail_folder_empty_card.dart:3]: Noticing a lot of StatefulWidget in this file for Widgets that don't really have much state. Wonder if we should add a PlausibleTrackingWidget wrapper that is stateful that we can use to wrap StatelessWidgets. Might clean up the codebase a little bit.

Comment (thiagocarvalhodev) [reply lib/pages/drive_detail/components/drive_detail_folder_empty_card.dart:3]: I liked that suggestion

**Comment by kunstmusik** [comment]
General note: tested on Firebase URL and found on mobile I could scroll to see empty state cards, but not on other breakpoints.

File: lib/pages/drive_detail/components/drive_detail_folder_empty_card.dart (hunk 1 above)
Comment (thiagocarvalhodev) [reply lib/pages/drive_detail/components/drive_detail_folder_empty_card.dart:3]: https://github.com/ardriveapp/ardrive-web/pull/1751/commits/23a2ed7a3ba9b9d79ccb660c02529418707bc277

**Review by kunstmusik — APPROVED** [review]

**Pull request merged**